spec/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.spec.js

```diff
@@ -0,0 +1,230 @@
+/* eslint-env jasmine, jquery */
```

Comment:
I think there's no jQuery in here so this could be removed?

Reply:
@andysellick the meta tag functions at the bottom of the file are using jQuery - should I replace them with vanilla JS?